scripts/k8s-etcd

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+# [This file is part of the zabbix-agent-osso package]
+# REQUIRES: kubectl, jq
+#
+#UserParameter=k8s-etcd.discovery, sudo /etc/zabbix/scripts/k8s-etcd --discover
+#UserParameter=k8s-etcd.status-json[*], sudo /etc/zabbix/scripts/k8s-etcd --get-values-etcd-status "$1" "$2" "$3" "$4"
+#UserParameter=k8s-etcd.health-json[*], sudo /etc/zabbix/scripts/k8s-etcd --get-values-etcd-health "$1" "$2" "$3" "$4"
+#
```

Comment:
Deze commented UserParameter= kunnen weg, denk ik? Die staan verbatim in de .conf die erbij hoort.